From 0f16d0d09112456f7c00e09347f0eae327ad8c4d Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Tue, 21 Jun 2016 12:21:16 -0400
Subject: [PATCH] erlang: enable compilation on mipsel

Erlang works fine on mipsel platforms so add it to the supported list.

Signed-off-by: Frank Hunleth <fhunleth@troodon-software.com>
---
 package/erlang/Config.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/erlang/Config.in b/package/erlang/Config.in
index 56cee54..11407db 100644
--- a/package/erlang/Config.in
+++ b/package/erlang/Config.in
@@ -3,7 +3,7 @@ config BR2_PACKAGE_ERLANG_ARCH_SUPPORTS
 	# see HOWTO/INSTALL.md for Erlang's supported platforms
 	# when using its native atomic ops implementation
 	default y if BR2_i386 || BR2_x86_64 || BR2_powerpc || \
-		BR2_sparc_v9 || BR2_arm || BR2_aarch64
+		BR2_sparc_v9 || BR2_arm || BR2_aarch64 || BR2_mipsel
 
 comment "erlang needs a toolchain w/ dynamic library"
 	depends on BR2_USE_MMU # fork()
-- 
2.5.0

